supplements/astral-adventurers-guide/items.xml

```diff
@@ -0,0 +1,682 @@
+		<description>
+			<p>Inside a Wildspace system, this portable arcane device automatically tracks the positions and movements of all suns, planets, moons, and comets within that system, projecting a display of all these bodies in the space above its current location. In that display, a white, pulsating pinprick of light marks the orrery’s location.</p>
+		</description>
+		<setters>
+			<set name="keywords">map, watch, compass, tracker</set>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Uncommon</set>
+			<set name="attunement">false</set>
+		</setters>
+	</element>
+
+	<!-- Misc. Items -->
+	<element name="Basic Fishing Equipment" type="Item" source="Astral Adventurer's Guide" id="ID_WOTC_AAG_ITEM_BASIC_FISHING_EQUIPMENT">
+		<description>
+			<p>Basic Fishing Equipment includes a pole, a line, a hook, and either a lure or some bait.</p>
+			<p class="indent">Fishing is a popular pastime in Wildspace and the Astral Sea, though this activity isn't possible aboard a ship that is moving faster than its flying speed. At the end of each hour spent fishing, a character can make a DC 15 Wisdom (Survival) check. A failed check indicates no fish is caught during that hour. On a successful check, roll a d10 and consult the Fishing table to determine the catch.</p>
+			<table>
+				<thead>
+					<tr><td>d10</td><td>Catch</td></tr>
+				</thead>
+				<tr><td>1-2</td><td>Tiny, inedible fish (a creature that consumes it is poisoned for 1 hour)</td></tr>
+				<tr><td>3-5</td><td>Tiny, edible fish (feeds one person)</td></tr>
+				<tr><td>6-8</td><td>Small, edible fish (feeds up to four people)</td></tr>
+				<tr><td>9</td><td>Hostile <b>space eel</b> (see <i>Boo's Astral Menagerie</i><!-- specify the page number -->; feeds up to twelve people)</td></tr>
+				<tr><td>10</td><td>Hostile <b>gray scavver</b> (see <i>Boo's Astral Menagerie</i><!-- specify the page number -->; feeds up to twenty people), some other creature of the DM's choosing, or an Tiny object of the DM's choosing</td></tr>
+			</table>
+		</description>
+		<setters>
+			<set name="category">Adventuring Gear</set>
+			<set name="type">Adventuring Gear</set>			
+			<set name="cost" currency="sp">1</set>
+			<set name="weight" lb="0">0 lb.</set>
+		</setters>
+	</element> 
+
+	<!-- Vehicles -->
+	<element name="Bombard" type="Item" source="Astral Adventurer's Guide" id="ID_WOTC_AAG_ITEM_BOMBARD">
+		<description>
+			<p>Bombards are built by giff. The major feature of each ship is an enormous cannon that fi res massive cannon balls capable of blowing other ships to smithereens. (The cannon is included in the cost of the ship.) A bombard can carry up to fourteen giant cannon balls, each of which weighs 10 tons. These cannon balls make up most of the weight of the ship’s cargo. A winch mounted on the aft deck is used to load the cannon balls on board.</p>
```

Comment:
typo of fi res should be fires
not sure how to do the type of comment which suggests these changes